tests/integration/targets/vmware_folder_template_from_vm/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+vcenter_port: 443
+run_on_simulator: false
+
+template_folder: e2e-qe
+
+vm: test-folder_template
+vm_cluster: "Eco-Cluster"
+vm_datacenter: "Eco-Datacenter"
```

Comment:
just fyi we have a [common vars yml file ](https://github.com/ansible-collections/vmware.vmware/blob/main/tests/integration/targets/group_vars.yml)you can import and use instead of writing the names